crates/qt-build-utils/src/lib.rs

```diff
@@ -484,6 +488,16 @@ impl QtBuild {
             );
         }
 
+        // If we have found at least one framework for the Qt modules then
+        // add the QT_INSTALL_LIBS as a framework link search path
+        //
+        // Note that leaving the kind empty should default to all,
+        // but this doesn't appear to find frameworks in all situations
+        // https://github.com/KDAB/cxx-qt/issues/885
+        if found_framework {
+            println!("cargo:rustc-link-search=framework={lib_path}");
```

Comment:
needs testing

Reply:
In a framework system i wonder if there ever are normal libraries in the `QT_INSTALL_LIBS` path or if there are only frameworks ? As the docs state that using `framework=` means that `Only search for macOS frameworks in this directory.` which would mean we could lose those?

Reply:
Appears that folder is mostly frameworks, only a few static libs (see the issue).